content/docs/buildpacks/go.md

```diff
@@ -0,0 +1,195 @@
+---
+title: "Go Buildpack"
+weight: 320
+menu:
+  main:
+    parent: "buildpacks"
+---
+
+# Go Buildpack
+
+The [Go CNB](https://github.com/paketo-buildpacks/go) supports several popular
+configurations for Go apps.
+
+To build your app locally with this CNB using the `pack` CLI, run:
+
+```
+pack build my-app --buildpack gcr.io/paketo-buildpacks/go
+```
+
+## <a id="specifying-go-version"></a> Specifying a Go Version
+
+The Go CNB allows you to specify a version of Go to use during deployment. This
+version can be specified via `buildpack.yml`. When specifying a version of Go,
+you must choose a version that is available within the buildpack. The supported
+versions can be found
+[here](https://github.com/paketo-buildpacks/go-dist/releases/latest).
+
+Specifying a version of Go is not required. In the case that is not specified,
+the buildpack will provide the default version, which can be seen in the
+buildpack.toml file.
+
+### <a id='buildpack-yml'></a> Using buildpack.yml
+
+To configure the buildpack to use Go v1.14.6 when deploying your app, include
+the values below in your `buildpack.yml` file:
+
+```
+---
+go:
+  version: 1.14.6
+```
+
+## <a id="specifying-build-flags"></a> Configuring Build Flags
+
+The `go build` command supports a number of flags that allow users to override defaults for more control over build configurations. By default, the buildpack sets the following build flags:
+
+* `-buildmode=pie`
+* `-mod=vendor` (if there is a go.mod file in the app source code)
+
+To set custom values for your build flags or override the defaults, add the values below to your `buildpack.yml` file:
+
+```
+---
+go:
+  ...
```

Comment:
```suggestion
```

Reply:
suggest removing ... so users can just copy anything in the code block and it would work

Reply:
Fair enough, done.